db.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const { MongoMemoryServer } = require('mongodb-memory-server');
+
+// A singleton to ensure we only start the database once
+// assign the MongoMemoryServer instance to mongoServer
+let mongoServer;
+const URL  ='mongodb+srv://bhumir:bhumi@cluster0.u0lqlrf.mongodb.net/?retryWrites=true&w=majority'
+
+
+const startDatabase = async () => {
+  // Your code here
+  await mongoose.connect(URL);
```

Comment:
Add error handling for the `mongoose.connect` and `mongoose.disconnect` methods. This will help you catch and handle any potential errors that might occur during the connection or disconnection process. [important]